### PR TITLE
actions: remove codecov as it breaks the CI

### PR DESCRIPTION
**Pull request by Wenzel**
Proposal: remove codecov as it breaks our CI and we don't know how to fix it.

### CODE REVIEW

File: codecov.yaml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-coverage:
-  status:
-    patch:
-      default:
-        target: auto
```
Comment (rageagainsthepc) [inline codecov.yaml:5]: According to the documentation this might help:
`target: 0%`

Comment (rageagainsthepc) [reply codecov.yaml:5]: It's the baseline of coverage that the commit must meet (Usually derived from master).

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -105,15 +105,6 @@ jobs:
           crate: grcov
           version: latest
           use-tool-cache: true
-      - name: generate coverage report
```
Comment (rageagainsthepc) [inline .github/workflows/build.yml:108]: If you are not uploading the coverage anymore there is no point in generating it in the first place.

File: codecov.yaml (hunk 1 above)
Comment (rageagainsthepc) [reply codecov.yaml:5]: Or even better: you can set a flag `ìnformational: true` This should make the CI pass no matter what.

**Comment by Wenzel** [comment]
already fixed by https://github.com/Wenzel/libmicrovmi/pull/102

**Pull request closed without merging**